Sources/AblyChat/RoomStatus.swift

```diff
@@ -31,3 +31,122 @@ public struct RoomStatusChange: Sendable {
         self.error = error
     }
 }
+
+internal actor DefaultRoomStatus: RoomStatus {
+    private(set) var current: RoomLifecycle
+    private(set) var error: ARTErrorInfo?
+
+    // TODO: how to clean this up?
+    private var stateChangeSubscriptions: [Subscription<RoomStatusChange>] = []
+    private let contributors: [ARTRealtimeChannelProtocol]
```

Comment:
Who are `contributors`? Participants of the chat room?

Reply:
Realtime channels that contribute to the room status — I’d suggest taking a look at the spec

Reply:
Requested access (in Jira)

Reply:
What did you request access for in Jira? The spec is in the spec repo — see link in commit message.

Reply:
Yeah, I've figured out already, thanks.